gameboard/game.js

```diff
@@ -0,0 +1,93 @@
+let ships = [
+  {
+    shipName: "Carrier",
+    size: 5,
+    amount: 1,
+  },
+  {
+    shipName: "	Battleship",
+    size: 4,
+    amount: 1,
+  },
+  {
+    shipName: "Destroyer",
+    size: 3,
+    amount: 2,
+  },
+  {
+    shipName: "Submarine",
+    size: 2,
+    amount: 2,
+  },
+  {
+    shipName: "Patrol Boat",
+    size: 1,
+    amount: 3,
+  },
+];
+
+let gameBoardWidth = 10;
+const gameBoard = document.querySelectorAll(".gameBoard__board-fieldsWrapper");
+const charWrapper = document.querySelectorAll(".gameBoard__charactersWrapper");
+const numbWrapper = document.querySelectorAll(".gameBoard__numbersWrapper");
+const playerNameField = document.querySelector(".player");
+const shipsList = document.querySelectorAll(".gameBoard__shipsWrapper-ships");
+
+//creating gameBoard
```

Comment:
Delete comment. If it doesn't explain a complicated and important thing / it doesn't annotate @TODO or it doesn't explain some hack you used because there was a need, then don't leave them in the code.